takhojin/app.js

```diff
@@ -0,0 +1,40 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+const { DataSource } = require("typeorm");
+
+dotenv.config();
+
+const myDataSource = new DataSource({
+  type: process.env.DB_CONNECTION,
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+});
+
+myDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized");
+});
+
+app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("combined"));
+
+app.get("/ping", (req, res) => {
+  res.json({ message: " pong " });
+});
```

Comment:
```suggestion
app.get("/ping", (req, res) => {
  return res.status(200).json({ message: " pong " });
});
```
http request중 GET에 맞는 status code를 반환할 수 있도록 백엔드에서 핸들링 할 수 있습니다.